dv-export/python/query-changes/command_handler.py

```diff
@@ -30,32 +47,48 @@ def execute_export_job(self, base_data_version: int, data_version: int) -> str:
         """Run a DV export job, wait for it to complete, and then return the export UUID."""
         return self.dv_manager.execute_export_job(base_data_version, data_version)
 
-    def process_query(self, export_uuid: str, query: dict[str, typing.Any]):
+    def process_query(self, export_uuid: str, query: dict[str, typing.Any], query_mode: QueryMode):
         """Process the query using filtering and save changes to the database."""
         analytic_object = query[SOURCE][ANALYTIC_OBJECT]
         logger.info(f"Analytic object to fetch changes: {analytic_object}")
 
         # Retrieve filter from query
-        filter_values = self._read_filter_values(analytic_object, export_uuid, query)
+        filter_name = self._get_filter_name(query)
+        filter_values = self._read_filter_values(export_uuid, analytic_object, filter_name) if filter_name else None
 
         # Retrieve analytic object changes
-        properties, changes_rows = self.changes_fetcher.list_changes(query, filter_values)
+        analytic_object_metadata, properties = self.changes_fetcher.get_analytic_object_metadata(analytic_object)
+
+        # Getting start and end for query changes depending on command mode
+        start_time_epoch, end_time_epoch = self._get_time_period_borders(export_uuid,
+                                                                         analytic_object_metadata,
+                                                                         query_mode)
+
+        # Fetch analytic object changes
+        changes_rows = self.changes_fetcher.execute_query_list(query, start_time_epoch, end_time_epoch,
```

Comment:
Nit: instead of `changes`, consider `change` for both the rows and the fetcher.

Reply:
Renamed changes_rows, filters_values -> change_rows, filter_values and etc.